rllib/policy/tests/test_sample_batch.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
```

Comment:
Make sure that all tests pass! This is a very big change to how SampleBatch works...